frontend/src/components/Auth/LoginForm.js

```diff
@@ -55,16 +60,16 @@ const LoginForm = ({ url, loginAction }) => {
             onChange={(e) => setFormData({ ...formData, password: e.target.value })}
           />
           {error.password ? (
-            <div className="ui pointing basic label" style={errorStyle}>
+            <div className={formClasses.fieldError}>
               {error.password}
             </div>
           ) : null}
         </div>
 
         {error.non_field_errors ? (
-          <div className="ui negative message" style={errorStyle}>
+          <center><div className={formClasses.error}>
             { error.non_field_errors }
-          </div>
+          </div><br></br><br></br></center>
```

Comment:
You can use ie. formClasses.lastError and define within property margin-bottom: 50px; text-align: center;